### PR TITLE
magento/devdocs#: Add updateCartItems query API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about the API errors of [updateCustomer](https://devdocs.magento.com/guides/v2.3/graphql/mutations/update-cart-items.html) mutation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/mutations/update-cart-items.html

## Links to Magento source code

- [Could not find cart item with id: XXX](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/UpdateCartItemsTest.php#L124)
- [Could not find a cart with ID "XXX"](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/UpdateCartItemsTest.php#L106)
- [Required parameter "cart_id" is missing.](https://github.com/magento/magento2/blob/2.3.5/app/code/Magento/QuoteGraphQl/Model/Resolver/UpdateCartItems.php#L72)
- [Required parameter "cart_items" is missing.](https://github.com/magento/magento2/blob/2.3.5/app/code/Magento/QuoteGraphQl/Model/Resolver/UpdateCartItems.php#L79)
- [Required parameter "quantity" for "cart_items" is missing.](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/UpdateCartItemsTest.php#L251)
- [The current user cannot perform operations on cart "XXX"](https://github.com/magento/magento2/blob/2.3.5/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/UpdateCartItemsTest.php#L176)

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/mutations/update-cart-items.md [hunk 1]
```diff
@@ -129,3 +129,14 @@ Attribute |  Data Type | Description
...
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find cart item with id: XXX` | The specified `input`.`cart_items`.`cart_item_id` value does not exist in the `quote_item` database table.
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` database table.
+`Required parameter "cart_id" is missing.` | The value specified in the `cart_id` argument is empty.
+`Required parameter "cart_items" is missing.` | The `cart_items` argument is empty or its value specified as a none array value.
+`Required parameter "quantity" for "cart_items" is missing.` | The required `input`.`cart_items`.`quantity` should be specified.
```
Comment (dmytro-ch) [inline src/guides/v2.3/graphql/mutations/update-cart-items.md:141]:
```suggestion
`Required parameter "quantity" for "cart_items" is missing.` | The required `input`.`cart_items`.`quantity` argument must be specified.
```

File: src/guides/v2.3/graphql/mutations/update-cart-items.md [hunk 2]
```diff
@@ -129,3 +129,14 @@ Attribute |  Data Type | Description
...
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find cart item with id: XXX` | The specified `input`.`cart_items`.`cart_item_id` value does not exist in the `quote_item` database table.
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` database table.
+`Required parameter "cart_id" is missing.` | The value specified in the `cart_id` argument is empty.
+`Required parameter "cart_items" is missing.` | The `cart_items` argument is empty or its value specified as a none array value.
```
Comment (rogyar) [inline src/guides/v2.3/graphql/mutations/update-cart-items.md:140]:
```suggestion
`Required parameter "cart_items" is missing.` | The `cart_items` argument is empty or its value specified as a non-array value.
```

File: src/guides/v2.3/graphql/mutations/update-cart-items.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/update-cart-items.md]: **fixed**

Thank you, @dmytro-ch

File: src/guides/v2.3/graphql/mutations/update-cart-items.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/update-cart-items.md]: **fixed**

Thank you, @rogyar

**Comment by keharper** [comment]
running tests

**Pull request merged**